cms/djangoapps/contentstore/tests/test_libraries.py

```diff
@@ -22,6 +22,12 @@
 from mock import Mock
 from opaque_keys.edx.locator import CourseKey, LibraryLocator
 from openedx.core.djangoapps.content.course_structures.tests import SignalDisconnectTestMixin
+from xmodule.x_module import STUDIO_VIEW
+import json
+from lxml import etree
+from StringIO import StringIO
+from HTMLParser import HTMLParser
```

Comment:
@StraNNiKK Once Jenkins runs, I suspect you'll get some quality error here for unused imports.